StellarDotnetSdk/Responses/TransactionResponse.cs

```diff
@@ -2,87 +2,181 @@
 using System.Collections.Generic;
 using System.Text.Json.Serialization;
 using StellarDotnetSdk.Memos;
-using StellarDotnetSdk.Responses.Effects;
-using StellarDotnetSdk.Responses.Operations;
 using StellarDotnetSdk.Responses.Results;
 
 namespace StellarDotnetSdk.Responses;
-#nullable disable
 
-public class TransactionResponse : Response, IPagingToken
+/// <summary>
+///     Represents a transaction on the Stellar network.
+///     Contains transaction metadata, results, and links to related resources.
+/// </summary>
+public sealed class TransactionResponse : Response, IPagingToken
 {
-    [JsonPropertyName("hash")] public string Hash { get; init; }
-
+    /// <summary>
+    ///     A unique identifier for this transaction.
+    /// </summary>
+    [JsonPropertyName("id")]
+    public required string Id { get; init; }
+
+    /// <summary>
+    ///     The hash of the transaction.
+    /// </summary>
+    [JsonPropertyName("hash")]
+    public required string Hash { get; init; }
+
+    /// <summary>
+    ///     The ledger sequence number in which this transaction was included.
+    /// </summary>
     [JsonPropertyName("ledger")]
-    public long Ledger { get; init; }
+    public required long Ledger { get; init; }
 
+    /// <summary>
+    /// An ISO 8601 formatted string of when the transaction was created.
+    /// </summary>
     [JsonPropertyName("created_at")]
-    public DateTime CreatedAt { get; init; }
+    public required string CreatedAt { get; init; }
```

Comment:
Should this not be a `DateTime`?
```suggestion
    public required DateTime CreatedAt { get; init; }
```

Reply:
It should either `DateTimeOffset` or `string`. `DateTime` omits the timezone part. I'm thinking about creating a separate follow-up PR to fix all these DateTime occurrences. Do you agree?